libs/island-ui/core/src/lib/AlertBanner/AlertBanner.tsx

```diff
@@ -143,6 +145,11 @@ export const AlertBanner: FC<AlertBannerProps> = ({
                 {description}
                 {description && link && ` `}
                 {link && <a href={link.href}>{link.title}</a>}
+                {link && link.tooltip && (
```

Comment:
The code looks fine so I'll approve. However I'd like to know if this change to the island-ui alert banner will also be kept up to date in Figma? Could we maybe add this as a story in storybook to show that this is possible?

Reply:
Thanks! I've now added a story for this and pinged the designers to notify them of this change and ask if they want to update the design.

Reply:
@RunarVestmann FYI, I've decided to revert these changes for now and work a bit more with the designers on this.